packages/tocco-ui/src/utilStyles/doesReuseAllValuesAsKeys.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Enforce that all values of a flat object are used as keys in another object.
+ * @param  {object} completeMap            A flat object with contains a full list of
+ *                                         all allowed keys defined as values.
+ * @param  {object} potentialIncompleteMap A flat object which is does not reuse all values as keys.
+ * @return {boolean}                       Return true if all values are reused as keys.
+ */
+
+const doesReuseAllValuesAsKeys = (completeMap, potentialIncompleteMap) => {
+  const values = Object.values(completeMap).sort()
+  const keys = Object.keys(potentialIncompleteMap).sort()
+
+  if (values.length !== keys.length) {
+    // eslint-disable-next-line no-console
+    console.warn(`Amount of identifier not equal (${values.toString()} !== ${keys.toString()}).`)
+    return false
+  }
+
+  if (!values.every(
+    (value, index) => {
+      return value === keys[index]
+    }
+  )) {
+    // eslint-disable-next-line no-console
+    console.warn(`At least on identifier does not match (${values.toString()} !== ${keys.toString()}).`)
+    return false
+  }
+
+  return true
+}
```

Comment:
Could we log the specific keys which are missing or should be removed? I reckon it's much more likely that I notice a log warning if a key is mentioned that I removed or added somewhere.